helm/charts/determined/templates/master-config.yaml

```diff
@@ -88,6 +93,13 @@ data:
       {{- if .Values.taskContainerDefaults.forcePullImage }}
       force_pull_image: {{ .Values.taskContainerDefaults.forcePullImage }}
       {{- end }}
+    {{ else if .Values.imageRegistry }}
+      {{- if not (eq .Values.imageRegistry "determinedai") }}
```

Comment:
This is actually redundant?

Reply:
Good point - before the changes I made to the path, this was necessary because I thought the paths were slightly different. Now they're identical after the registry path.